src/components/ButtonWithColor/ButtonWithColor.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import './ButtonWithColor.css';
+
+interface ButtonWithColorProps {
+    color?: string;
```

Comment:
will use it, but it is not ready, this button  is can only customize the context now.